script.js

```diff
@@ -0,0 +1,442 @@
+const myobj={};
+
+function appendToInput(value) {
+    const textInput = document.getElementById('textInput');
+
+    const s = Array.from(textInput.value);
+
+    if(value !== "+" && value !== "-" && value !== "*" && value !== "/" && value !== "^" &&
+      value !== "sqrt(" && value !== "sin(" && value !== "cos(" && value !== "tan(" && value !== ")"  && value !== "pi"  && value !== "e")
+      {
+        const v = Number(value);
+        if(s[s.length-1] === ")")
```

Comment:
Change the if else to switch case, create a separate function for this functionality!